cloudman/data/update_keycloak.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/sh
+
+# abort if any command fails
+set -e
+username="admin"
+password="{{ .Values.keycloak.keycloak.password }}"
+
+# get auth token
+token=$(curl -k -s -d "client_id=admin-cli" -d "username=admin" -d "password=$password" -d "grant_type=password" \
+       "https://{{ .Values.global.domain }}/auth/realms/master/protocol/openid-connect/token" | jq -r '.access_token')
+
+# get admin user id
+user_id=$(curl -k -s -H "Content-Type: application/json" -H "Authorization: bearer $token" https://{{ .Values.global.domain }}/auth/admin/realms/master/users/?username=admin | \
+          jq -r '.[] | select(.username=="admin") | .id')
+
+# update admin user info
+updated_user=$(curl -k -s -H "Content-Type: application/json" -H "Authorization: bearer $token" https://{{ .Values.global.domain }}/auth/admin/realms/master/users/$user_id | \
+          jq -r '.firstName="Alex" | .lastName="ITNEW" | .email="newtest@me.org"')
+
+# Save new info
+curl -k -X PUT -H "Content-Type: application/json" -H "Authorization: bearer $token" https://{{ .Values.global.domain }}/auth/admin/realms/master/users/$user_id -d "$updated_user"
+
+# Add Brute Force Detection to Master realm
+curl -k -X PUT -H "Content-Type: application/json" -H "Authorization: bearer $token" https://{{ .Values.global.domain }}/auth/admin/realms/master -d '{"bruteForceProtected": true, "failureFactor": 5, "maxFailureWaitSeconds": 1800, "minimumQuickLoginWaitSeconds": 300}'
```

Comment:
Making these helm parameters would be good I think.

Reply:
Also, when moving to init-container, we'll need a check to make sure `bruteForceProtected` is not already true.

Reply:
Done in https://github.com/almahmoud/cloudman-helm/commit/f76baa9816521e35c6f49003998885377e5c97de and https://github.com/CloudVE/cloudlaunch-helm/pull/4